### PR TITLE
docs(CalendarAndDateRangePicker): Add AllowReverse and ShowOtherMonth…

### PR DESCRIPTION
**Pull request by NansiYancheva**
Related to https://github.com/telerik/blazor/issues/8689

### CODE REVIEW

File: components/calendar/overview.md [hunk 1]
```diff
@@ -92,11 +88,12 @@ The Blazor Calendar provides various parameters that allow you to configure the
 | `RangeEnd` | `DateTime` | The selected end date when [range selection]({%slug components/calendar/selection%}#range-selection-mode) is enabled. Supports two-way binding. |
 | `SelectedDates` | `List<DateTime>` | The selected dates when [multiple selection]({%slug components/calendar/selection%}#multiple-selection-mode) is used. |
 | `SelectionMode` | `CalendarSelectionMode` enum <br /> (`Single`) | The [selection mode]({%slug components/calendar/selection%}) of the calendar. |
+| `ShowWeekNumbers` | `bool` | Sets if the Calendar will display week numbers according to the [ISO-8601 format](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.isoweek.getweekofyear). Note that the [ISO week number may differ from the conventional .NET week number](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.calendar.getweekofyear). |
+| `ShowOtherMonthDays` | `bool` <br /> (`true`) | Defines whether the leading and trailing days from other months are visible within the current month. |
```
Comment (dimodi) [inline components/calendar/overview.md:92]: The other month's days are not actually displayed IN / WITHIN the current month.

```suggestion
| `ShowOtherMonthDays` | `bool` <br /> (`true`) | Defines whether the leading and trailing days from other months are visible in the current month view. |
```

File: components/calendar/selection.md [hunk 2]
```diff
@@ -135,20 +135,29 @@ With multiple selection mode, to get the user selection, use the `SelectedDates`
 
 With range selection mode, you have two options to get the user choice:
 
-* two-way binding for the `RangeStart` (representing the first date of the selection) and the `RangeEnd` (the last date of the selection) parameters.
-* Handling the [RangeStartChanged and RangeEndChanged events]({%slug components/calendar/events%}#rangestartchanged-and-rangeendchanged)
+* Two-way binding for the `RangeStart` (representing the first date of the selection) and the `RangeEnd` (the last date of the selection) parameters.
+* Handling the [RangeStartChanged and RangeEndChanged events]({%slug components/calendar/events%}#rangestartchanged-and-rangeendchanged).
```
Comment (dimodi) [inline components/calendar/selection.md:139]:
```suggestion
* Configure two-way binding for the `RangeStart` and the `RangeEnd` parameters. They represent the first and last date of the selection.
* Handle the [RangeStartChanged and RangeEndChanged events]({%slug components/calendar/events%}#rangestartchanged-and-rangeendchanged).
```

File: components/calendar/selection.md [hunk 3]
```diff
@@ -135,20 +135,29 @@ With multiple selection mode, to get the user selection, use the `SelectedDates`
...
+
+>caption Range selection with two-way binding and AllowReverse
 
 ````CSHTML
 @* This example shows how to handle Range selection through two-way binding *@
 
+<TelerikCheckBox Id="myCheckBox" @bind-Value="@AllowReverse" />
+<label for="myCheckBox">@(AllowReverse ? "Allowed reverse selection" : "Not allowed reverse selection")</label>
+
+<br/>
```
Comment (dimodi) [inline components/calendar/selection.md:152]:
```suggestion
<br />
```

File: components/calendar/selection.md [hunk 4]
```diff
@@ -161,6 +170,7 @@ With range selection mode, you have two options to get the user choice:
     public DateTime Date { get; set; } = DateTime.Now.AddDays(-5);
     public DateTime RangeStart { get; set; } = DateTime.Now.Date;
     public DateTime RangeEnd { get; set; } = DateTime.Now.AddDays(15).Date;
+    public bool AllowReverse { get; set; }
```
Comment (dimodi) [inline components/calendar/selection.md:173]: Switch all accessors to `private`

File: components/daterangepicker/overview.md [hunk 5]
```diff
@@ -15,11 +15,12 @@ The <a href="https://www.telerik.com/blazor-ui/daterange-picker" target="_blank"
 ## Creating Blazor Date Range Picker
 
 1. Use the `TelerikDateRangePicker` tag to add the component to your razor page.
-1. Bind its `StartValue` and `EndValue` parameters to `DateTime` objects
-1. Optionally, provide custom `Format`, `Min` and `Max` values 
+1. Bind its `StartValue` and `EndValue` parameters to `DateTime` objects.
+1. Optionally, provide custom `Format`, `Min` and `Max` values.
```
Comment (dimodi) [inline components/daterangepicker/overview.md:19]: I think we use this formatting more: 

```suggestion
1. (optional) Provide custom `Format`, `Min` and `Max` values.
```

**Review by dimodi — APPROVED** [review]

**Review by ikoevska — APPROVED** [review]

**Pull request merged**